examples/soft-kmeans/probability.py

```diff
@@ -2,13 +2,14 @@
 import numpy as np
 from scipy.special import logsumexp, softmax
 
+from chainsail_helpers.pdf import PosteriorPDF
 
 # `data` is of dimensions (n_points, n_dim)
 path = os.path.dirname(os.path.abspath(__file__))
 data = np.loadtxt(os.path.join(path, "data.txt"), ndmin=2)
 
 
-class Pdf:
+class Pdf(PosteriorPDF):
```

Comment:
There is a class `PDF` and here class `Pdf` inherits after `PosteriorPDF`. I think that we need to either change `PDF` to something like `GeneralPDF` or name `Pdf` differently to avoid confusion.

Reply:
:+1: addressed in https://github.com/tweag/chainsail-resources/pull/16/commits/de9aec436bebdab271e8ee08bfd6bc944c71ebb3.